### PR TITLE
docs: add python write demo

### PR DESCRIPTION
**Pull request by Reid00**
doc effect only

### CODE REVIEW

**Comment by wey-gu** [comment]
There is actually vertex write examples in L176, could you please combine your demo example (with edge) in that section?

https://github.com/vesoft-inc/nebula-spark-connector/blob/dd535bfb4ecee8e01f81e56cb38147c09d2edcf2/README.md?plain=1#L176-L188

**Comment by Reid00** [comment]
@wey-gu updated.

**Review by wey-gu — DISMISSED** [review]
LGTM Thanks!!!

File: README.md [hunk 1]
```diff
@@ -206,6 +206,44 @@ df.write.format("com.vesoft.nebula.connector.NebulaDataSource").option(
...
+    .option("metaAddress", "metad0:9559")\
+    .option("graphAddress", "graphd:9669")\
+    .option("user", "root")\
+    .option("passwd", "nebula")\
+    .option("type", "edge")\
+    .option("spaceName", "basketballplayer")\
+    .option("label", "server")\
+    .option("srcVertexField", "srcid")\
+    .option("dstVertexField", "dstid")\
+    .option("rankFiled", "")\
```
Comment (Nicole00) [inline README.md:224]: pr https://github.com/vesoft-inc/nebula-spark-connector/pull/58 has been merged, could you please update the rankfiled to rankfield?

File: README.md [hunk 2]
```diff
@@ -206,6 +206,44 @@ df.write.format("com.vesoft.nebula.connector.NebulaDataSource").option(
...
```
Comment (Nicole00) [inline README.md:243]: ditto

File: README_CN.md [hunk 3]
```diff
@@ -206,6 +207,44 @@ df.write.format("com.vesoft.nebula.connector.NebulaDataSource").option(
...
+    .option("metaAddress", "metad0:9559")\
+    .option("graphAddress", "graphd:9669")\
+    .option("user", "root")\
+    .option("passwd", "nebula")\
+    .option("type", "edge")\
+    .option("spaceName", "basketballplayer")\
+    .option("label", "server")\
+    .option("srcVertexField", "srcid")\
+    .option("dstVertexField", "dstid")\
+    .option("rankFiled", "")\
```
Comment (Nicole00) [inline README_CN.md:225]: ditto

File: README_CN.md [hunk 4]
```diff
@@ -206,6 +207,44 @@ df.write.format("com.vesoft.nebula.connector.NebulaDataSource").option(
...
```
Comment (Nicole00) [inline README_CN.md:244]: ditto

File: nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala [hunk 5]
```diff
@@ -220,7 +220,7 @@ object NebulaOptions {
   val TIMEOUT: String            = "timeout"
   val CONNECTION_RETRY: String   = "connectionRetry"
   val EXECUTION_RETRY: String    = "executionRetry"
-  val RATE_TIME_OUT: String      = "reteTimeOut"
+  val RATE_TIME_OUT: String      = "rateTimeOut"
```
Comment (Nicole00) [inline nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala:223]: good catch, thanks

**Comment by wey-gu** [comment]
ping @Reid00

**Comment by Reid00** [comment]
pls take a look, I update all rankField in README.md

**Review by wey-gu — APPROVED** [review]
thanks!!

**Pull request merged**